torch/nn/parallel/distributed.py

```diff
@@ -800,33 +835,32 @@ def forward(self, *inputs, **kwargs):
             else:
                 output = self.module(*inputs, **kwargs)
 
+            disable = False
```

Comment:
why is this flag needed?

Reply:
Thanks for flagging, accidentally send debug code. Removing.